### PR TITLE
Updates and clarity to data connectors

### PR DESCRIPTION
**Pull request by lukekim**

### CODE REVIEW

File: spiceaidocs/docs/components/data-connectors/index.md [hunk 1]
```diff
@@ -11,62 +11,66 @@ Data Connectors provide connections to databases, data warehouses, and data lake
...
-| `sharepoint`    | SharePoint    | Alpha  |                                     | `full`           | ❌               | ✅                |
-| `snowflake`     | Snowflake     | Alpha  | Arrow                               | `full`           | ❌               | ❌                |
-| `spiceai`       | Spice.ai      | Alpha  | Arrow Flight                        | `append`, `full` | ✅               | ❌                |
-| `s3`            | S3            | Alpha  | Parquet, CSV                        | `full`           | ❌               | ✅                |
-| `abfs`            | Azure BlobFS            | Alpha  | Parquet, CSV                        | `full`           | ❌               | ✅                |
-| `sharepoint`    | SharePoint    | Alpha  |                                     | `full`           | ❌               | ✅                |
-| `spark`         | Spark         | Alpha  | Spark Connect                       | `full`           | ❌               | ❌                |
+| Name            | Description   | Status | Protocol/Format                     | Refresh Modes               | Supports Ingestion | Supports Documents |
```
Comment (peasee) [inline spiceaidocs/docs/components/data-connectors/index.md]: what does "Supports Ingestion" mean?

File: spiceaidocs/docs/components/data-connectors/index.md [hunk 2]
```diff
@@ -11,62 +11,66 @@ Data Connectors provide connections to databases, data warehouses, and data lake
...
+| --------------- | ------------- | ------ | ----------------------------------- | --------------------------- | ------------------ | ------------------ |
+| `abfs`          | Azure BlobFS  | Alpha  | Parquet, CSV                        | `append`, `full`            | Roadmap            | ✅                 |
```
Comment (peasee) [inline spiceaidocs/docs/components/data-connectors/index.md:16]: what does "Roadmap" for "Supports Ingestion" mean?

File: spiceaidocs/docs/components/data-connectors/index.md [hunk 3]
```diff
@@ -11,62 +11,66 @@ Data Connectors provide connections to databases, data warehouses, and data lake
...
+| `clickhouse`    | Clickhouse    | Alpha  |                                     | `append`, `full`            | ❌                 | ❌                 |
+| `databricks`    | Databricks    | Beta   | Spark Connect <br/> S3 / Delta Lake | `append`, `full`            | Roadmap            | ❌                 |
+| `debezium`      | Debezium      | Alpha  | CDC, Kafka                          | `append`, `full`, `changes` | ❌                 | ❌                 |
+| `delta_lake`    | Delta Lake    | Beta   | Delta Lake                          | `append`, `full`            | Roadmap            | ❌                 |
+| `dremio`        | Dremio        | Alpha  | Arrow Flight SQL                    | `append`, `full`            | ❌                 | ❌                 |
+| `file`          | File          | Beta   | Parquet, CSV                        | `append`, `full`            | Roadmap            | ✅                 |
```
Comment (peasee) [inline spiceaidocs/docs/components/data-connectors/index.md]: Is the file connector Beta? I don't recall us testing it from the old Beta criteria.

File: spiceaidocs/docs/components/data-connectors/index.md [hunk 4]
```diff
@@ -11,62 +11,66 @@ Data Connectors provide connections to databases, data warehouses, and data lake
...
+| `flightsql`     | FlightSQL     | Beta   | Arrow Flight SQL                    | `append`, `full`            | ❌                 | ❌                 |
+| `ftp`, `sftp`   | FTP/SFTP      | Alpha  | Parquet, CSV                        | `append`, `full`            | ❌                 | ✅                 |
+| `github`        | GitHub        | Beta   | GraphQL, REST                       | `append`, `full`            | ❌                 | ❌                 |
```
Comment (peasee) [inline spiceaidocs/docs/components/data-connectors/index.md]: Similarly. We don't yet have criteria to cover Beta for connectors like GitHub, Sharepoint, etc.

File: spiceaidocs/docs/components/data-connectors/index.md [hunk 5]
```diff
@@ -11,62 +11,66 @@ Data Connectors provide connections to databases, data warehouses, and data lake
...
+| `graphql`       | GraphQL       | Alpha  | GraphQL                             | `append`, `full`            | ❌                 | ❌                 |
+| `http`, `https` | HTTP(s)       | Alpha  | Parquet, CSV                        | `append`, `full`            | ❌                 | ❌                 |
+| `mssql`         | MS SQL Server | Alpha  | Tabular Data Stream (TDS)           | `append`, `full`            | ❌                 | ❌                 |
+| `mysql`         | MySQL         | Beta   |                                     | `append`, `full`            | Roadmap            | ❌                 |
+| `odbc`          | ODBC          | Alpha  |                                     | `append`, `full`            | ❌                 | ❌                 |
```
Comment (peasee) [inline spiceaidocs/docs/components/data-connectors/index.md]: ODBC is in Beta? It passed the beta hardening criteria IIRC

File: spiceaidocs/docs/components/data-connectors/index.md (hunk 1 above)
Comment (lukekim) [inline spiceaidocs/docs/components/data-connectors/index.md]:
```suggestion
| Name            | Description   | Status | Protocol/Format                     | Refresh Modes               | Supports [Ingestion](https://docs.spiceai.org/features/data-ingestion) | Supports Documents |
```

File: spiceaidocs/docs/components/data-connectors/index.md [hunk 6]
```diff
@@ -11,62 +11,66 @@ Data Connectors provide connections to databases, data warehouses, and data lake
...
+| `abfs`          | Azure BlobFS  | Alpha  | Parquet, CSV                        | `append`, `full`            | Roadmap            | ✅                 |
+| `clickhouse`    | Clickhouse    | Alpha  |                                     | `append`, `full`            | ❌                 | ❌                 |
+| `databricks`    | Databricks    | Beta   | Spark Connect <br/> S3 / Delta Lake | `append`, `full`            | Roadmap            | ❌                 |
+| `debezium`      | Debezium      | Alpha  | CDC, Kafka                          | `append`, `full`, `changes` | ❌                 | ❌                 |
+| `delta_lake`    | Delta Lake    | Beta   | Delta Lake                          | `append`, `full`            | Roadmap            | ❌                 |
+| `dremio`        | Dremio        | Alpha  | Arrow Flight SQL                    | `append`, `full`            | ❌                 | ❌                 |
+| `file`          | File          | Beta   | Parquet, CSV                        | `append`, `full`            | Roadmap            | ✅                 |
```
Comment (lukekim) [inline spiceaidocs/docs/components/data-connectors/index.md]:
```suggestion
| `file`          | File          | Alpha   | Parquet, CSV                        | `append`, `full`            | Roadmap            | ✅                 |
```

File: spiceaidocs/docs/components/data-connectors/index.md [hunk 7]
```diff
@@ -11,62 +11,66 @@ Data Connectors provide connections to databases, data warehouses, and data lake
...
+| `debezium`      | Debezium      | Alpha  | CDC, Kafka                          | `append`, `full`, `changes` | ❌                 | ❌                 |
+| `delta_lake`    | Delta Lake    | Beta   | Delta Lake                          | `append`, `full`            | Roadmap            | ❌                 |
+| `dremio`        | Dremio        | Alpha  | Arrow Flight SQL                    | `append`, `full`            | ❌                 | ❌                 |
+| `file`          | File          | Alpha   | Parquet, CSV                        | `append`, `full`            | Roadmap            | ✅                 |
+| `flightsql`     | FlightSQL     | Beta   | Arrow Flight SQL                    | `append`, `full`            | ❌                 | ❌                 |
+| `ftp`, `sftp`   | FTP/SFTP      | Alpha  | Parquet, CSV                        | `append`, `full`            | ❌                 | ✅                 |
+| `github`        | GitHub        | Beta   | GraphQL, REST                       | `append`, `full`            | ❌                 | ❌                 |
```
Comment (lukekim) [inline spiceaidocs/docs/components/data-connectors/index.md]:
```suggestion
| `github`        | GitHub        | Alpha   | GraphQL, REST                       | `append`, `full`            | ❌                 | ❌                 |
```

File: spiceaidocs/docs/components/data-connectors/index.md [hunk 8]
```diff
@@ -11,62 +11,66 @@ Data Connectors provide connections to databases, data warehouses, and data lake
...
+| `ftp`, `sftp`   | FTP/SFTP      | Alpha  | Parquet, CSV                        | `append`, `full`            | ❌                 | ✅                 |
+| `github`        | GitHub        | Alpha   | GraphQL, REST                       | `append`, `full`            | ❌                 | ❌                 |
+| `graphql`       | GraphQL       | Alpha  | GraphQL                             | `append`, `full`            | ❌                 | ❌                 |
+| `http`, `https` | HTTP(s)       | Alpha  | Parquet, CSV                        | `append`, `full`            | ❌                 | ❌                 |
+| `mssql`         | MS SQL Server | Alpha  | Tabular Data Stream (TDS)           | `append`, `full`            | ❌                 | ❌                 |
+| `mysql`         | MySQL         | Beta   |                                     | `append`, `full`            | Roadmap            | ❌                 |
+| `odbc`          | ODBC          | Alpha  |                                     | `append`, `full`            | ❌                 | ❌                 |
```
Comment (lukekim) [inline spiceaidocs/docs/components/data-connectors/index.md]:
```suggestion
| `odbc`          | ODBC          | Beta  |                                     | `append`, `full`            | ❌                 | ❌                 |
```

**Review by peasee — APPROVED** [review]

**Pull request merged**